app/default_overrides.go

```diff
@@ -162,7 +162,7 @@ func (govModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 	oneWeek := day * 7
 	twoWeeks := oneWeek * 2
 
-	genState.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(BondDenom, sdk.NewInt(1_000_000_000))) // 1000 TIA
+	genState.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(BondDenom, sdk.NewInt(100_000_000_000))) // 10,000 TIA
```

Comment:
[blocking] sorry you had this right originally before my suggestion

```suggestion
	genState.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(BondDenom, sdk.NewInt(10_000_000_000))) // 10,000 TIA
```